### ISSUE TITLE
How can I add a road styling to a new project? 

### ISSUE THREAD

**Issue by TriniHam** [tilemill-project/tilemill#2756]

**Comment by TriniHam** [comment]
> I'd suggest starting with a sample project like OSM Bright https://github.com/mapbox/osm-bright/, which has several styles already defined, and pulls in different data sources.
> 
> The TileMill documentation has lots of info on styling: https://tilemill-project.github.io/tilemill/docs/crashcourse/styling/
> 
> For the example project/maps you showed above, to change the background ocean blue to your desired color, just change the hex color for `background-color`. Change the country color by changing `polygon-color`.

Yes I tried those styling and I saw the OSM Bright for [windows](https://gist.github.com/jasonwoo/7794945) but getting the postgres/PostGIS to install is pain. The guy used OpenGeo Suite 4.0.1 but his [link ](http://boundlessgeo.com/solutions/opengeo-suite/download/) isn't working. I don't want to download something else and my system gets malware. Do you have any suggestions for this ?

**Comment by csytsma** [comment]
Unfortunately, we're not familiar with installing on Windows. Perhaps someone familiar with Windows can chime in.

**Comment by prusswan** [comment]
@TriniHam PostGIS bundle for PostgreSQL only contains the PostGIS extension, you need to install the required PostgreSQL version first (which is why the PostGIS installer is looking for it). You can probably use the EDB installers (https://www.enterprisedb.com/downloads/postgres-postgresql-downloads) that will give the options to install PostGIS and PgAdmin etc.
